types/wx-server-sdk/index.d.ts

```diff
@@ -0,0 +1,338 @@
+    }
+    interface MultiLineString {
+        (polygons: LineString[]): MultiLineString;
+    }
+    interface MultiLineString {
+        type: "MultiLineString";
+        coordinates: PointCoordinates[][];
+    }
+    interface MultiPolygon {
+        (polygons: Polygon[]): MultiPolygon;
+    }
+    interface MultiPolygon {
+        type: "MultiPolygon";
+        coordinates: PointCoordinates[][][];
+    }
+
+    // interface GeoJSON<T> {
+    //     type: T;
+    //     coordinates: [];
+    // }
+
+    interface CreateCollectionSuccess {
+        errMsg: string;
+    }
+
+    interface Aggregate {
+        addFields(fieldObj: { [fieldName: string]: any }): Aggregate;
+        bucket(bucketObj: {
+            groupBy: any;
+            boundaries: any[];
+            default?: any;
+            output?: object;
+        }): Aggregate;
+        bucketAuto(bucketObj: {
+            groupBy: any;
+            buckets: number;
+            granularity?: any;
+            output?: object;
+        }): Aggregate;
+        count(expr: string): any;
+        geoNear(geoNearObj: {
+            near: Point;
+            spherical: true;
+            limit?: number;
+            maxDistance?: number;
+            minDistance?: number;
+            query?: object;
+            distanceMultiplier?: number;
+            distanceField: string;
+            includeLocs?: string;
+            key?: string;
+        }): Aggregate;
+        group(groupObj: { _id: any; [fieldName: string]: any }): Aggregate;
+        limit(limitRecords: number): any;
+        match(matchObj: { [fieldName: string]: any }): Aggregate;
+        project(projectObj: { [fieldName: string]: any }): Aggregate;
+        replaceRoot(replaceRootObj: { newRoot: any }): Aggregate;
+        sample(replaceRootObj: { size: number }): Aggregate;
+        skip(skipNum: number): any;
+        sort(replaceRootObj: { [fieldName: string]: 1 | -1 }): Aggregate;
+        sortByCount(fieldName: string): Aggregate;
+        unwind(unwindObj: {
+            path: string;
+            includeArrayIndex?: string;
+            preserveNullAndEmptyArrays?: boolean;
+        }): Aggregate;
+        end(): void;
+    }
+    // type
+    interface AggregationOperators {
+        abs(operand: number): number;
+        add(...operand: any[]): any;
+        addToSet(expression: string): any;
+        allElementsTrue(expression: [string]): boolean;
+        and(expression: boolean[]): boolean;
+        lt(expression: string, value: number): boolean;
+        lte(expression: string, value: number): boolean;
+        anyElementTrue(expression: [string]): boolean;
+        arrayElemAt(expression: [string, number]): any;
+        arrayToObject(expression: string): object;
+        // arrayToObject(expression: [string, any][]): object
+        // arrayToObject(expression: {k: string; v: any}[]): object
+        avg(expression: string): number;
+    }
+}
+
+// tslint:disable-next-line:export-just-namespace
```

Comment:
Please do not disable this rule. Export your module contents according to the rule's guidance:
https://github.com/Microsoft/dtslint/blob/master/docs/export-just-namespace.md

Reply:
thanks for your advice, the disabe rule has been removed